packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -24,7 +25,8 @@ export interface MenuButtonItemProps extends ContextualMenuItemProps {
   showSubmenu?: boolean;
 }
 
-export interface MenuButtonProps extends ButtonProps {
+export interface MenuButtonProps extends Omit<ButtonProps, 'icon'> {
+  startIcon?: IconSourcesType;
```

Comment:
The old button also allowed for `endIcon`, could we add that too?

This page has a full diff of the props between old and new button: https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/MIGRATION.md#props-no-longer-supported-with-an-equivalent-functionality-in-v1-button